### DISCUSSION TITLE
Draw a Circle in Paint

### DISCUSSION THREAD

**Discussion by io-world** [TestPlant/sensetalk-solutions#8] [Challenge Suggestions]
Learn how to use the math functions to Draw a Circle in Paint.

**Comment by CosmicKetchup** [comment]
<img src="https://media.giphy.com/media/90F8aUepslB84/giphy.gif">

Love this suggestion. Are there any other requirements? Has this actually been a question posed to you by a customer?

**Reply by io-world** [reply]
it was from a customer, specifically around sliding a circular Dial with a number in the middle. learning drag and drop in a circular fashion was fun to learn.
